VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -43,6 +43,7 @@
     public static final String EXTRA_DISCUSSION_THREAD = "discussion_thread";
     public static final String EXTRA_DISCUSSION_COMMENT = "discussion_comment";
     public static final String EXTRA_DISCUSSION_TOPIC_ID = "discussion_topic_id";
+    public static final String EXTRA_DISCUSSION_HAS_TOPIC_NAME = "discussion_has_topic_name";
```

Comment:
All our extra keys are listed in `Router.java` lets keep it that way, even if the usage of a key is specific to a class (it may get used in another class in the future).